.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ locals:
 executors:
   go:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/golang:1.13.15
+      - image: docker.mirror.hashicorp.services/circleci/golang:1.18
```

Comment:
This should also be updated to 1.19

Reply:
the lastest hashicorp mirror image is now 1.17.5
when I made this PR it was >= 1.18